### ISSUE TITLE
[BUG] Delete First Project Lead to A Crush

### ISSUE THREAD

**Issue by wtdry** [Leantime/leantime#2381]
*If you have problems with the installation please use our [community forum](https://community.leantime.io)*

**What set up are you using**
* [x] Self Hosted

**Describe the bug**
Delete the first created project Leads to a crush

**To Reproduce**
1. Install via docker compose file:
```
version: '3.3'
services:
  leantime_db:
    image: mysql:8.0
    container_name: mysql_leantime
    volumes:
      - db_data:/var/lib/mysql
    restart: unless-stopped
    env_file: ./.env                                        # Environment file with settings
    networks:
      - leantime-net
    command: --character-set-server=UTF8MB4 --collation-server=UTF8MB4_unicode_ci
  leantime:
    image: leantime/leantime:latest
    container_name: leantime
    restart: unless-stopped
    env_file: ./.env                                        # Environment file with settings
    networks:
      - leantime-net
    labels:
      traefik.enable: true
      traefik.http.routers.lean.rule: Host(`lean.mydomain.com`, `lean.local`)
      traefik.http.routers.lean.entrypoints: http
    volumes:
      - public_userfiles:/var/www/html/public/userfiles     # Volume to store public files, logo etc
      - userfiles:/var/www/html/userfiles                   # Volume to store private user uploaded files
    ports:
      - "${LEAN_PORT}:80"                                          # The port to expose and access Leantime
    depends_on:
      - leantime_db                                         # Don't start Leantime unless leantime_db is running
volumes:
  db_data:
  userfiles:
  public_userfiles:

networks:
  leantime-net:
```
Env File:
```
LEAN_PORT = 3035                                # The port to expose and access Leantime
LEAN_APP_URL = 'https://lean.mydomain.com'                             # Base URL, only needed for subfolder installation
LEAN_APP_DIR = ''                             # Base of application without trailing slash (used for cookies), e.g, /leantime
LEAN_DEBUG = 1                                 # Debug flag
MYSQL_ROOT_PASSWORD = 'randompass'                 # Database password
MYSQL_DATABASE = 'leantime'                         # Database name
MYSQL_USER = 'lean'                                 # Database username
MYSQL_PASSWORD = 'randompass'                      # Database password
LEAN_DB_HOST = 'mysql_leantime'                    # Database host 
LEAN_DB_USER = 'lean'                              # Database username (needs to be the same as MYSQL_USER)
LEAN_DB_PASSWORD = 'randompass'                   # Database password (needs to be the same as MYSQL_PASSWORD)
LEAN_DB_DATABASE = 'leantime'                      # Database name (needs to be the same as MYSQL_DATABASE)
LEAN_DB_PORT = '3306'                              # Database port
LEAN_SITENAME = 'mysitename'                         # Name of your site, can be changed later
LEAN_LANGUAGE = 'en-US'                            # Default language
LEAN_DEFAULT_TIMEZONE = 'Asia/Hong_Kong'      # Set default timezone
LEAN_ENABLE_MENU_TYPE = true                      # Enable to specifiy menu on a project by project basis
LEAN_SESSION_PASSWORD = 'somepass'  #Salting sessions. Replace with a strong password
LEAN_SESSION_EXPIRATION = 28800                    # How many seconds after inactivity should we logout?  28800seconds = 8hours
LEAN_LOG_PATH = null                                # Default Log Path (including filename), if not set /logs/error.log will be used
LEAN_LOGO_PATH = '/images/logo.svg'                # Default logo path, can be changed later
LEAN_PRINT_LOGO_URL = '/images/logo.jpg'           # Default logo URL use for printing (must be jpg or png format)
LEAN_DEFAULT_THEME = 'default'                     # Default theme
LEAN_PRIMARY_COLOR = '#1b75bb'                     # Primary Theme color
LEAN_SECONDARY_COLOR = '#81B1A8'                   # Secondary Theme Color
LEAN_USER_FILE_PATH = 'userfiles/'                 # Local relative path to store uploaded files (if not using S3)
LEAN_DB_BACKUP_PATH = 'backupdb/'                  # Local relative path to store backup files, need permission to write
LEAN_EMAIL_RETURN = 'admin@mydomain.com'                             # Return email address, needs to be valid email address format
LEAN_EMAIL_USE_SMTP = true                        # Use SMTP? If set to false, the default php mail() function will be used
LEAN_EMAIL_SMTP_HOSTS = 'smtp.domain.com'                         # SMTP host
LEAN_EMAIL_SMTP_AUTH = true                        # SMTP authentication required
LEAN_EMAIL_SMTP_USERNAME = 'noreply@mydomain.com'                      # SMTP username
LEAN_EMAIL_SMTP_PASSWORD = 'somepass'                      # SMTP password
LEAN_EMAIL_SMTP_AUTO_TLS = true                         # SMTP Enable TLS encryption automatically if a server supports it
LEAN_EMAIL_SMTP_SECURE = 'SSL'                        # SMTP Security protocol (usually one of: TLS, SSL, STARTTLS)
LEAN_EMAIL_SMTP_SSLNOVERIFY = false                # SMTP Allow insecure SSL: Don't verify certificate, accept self-signed, etc.
LEAN_EMAIL_SMTP_PORT = '587'                          # Port (usually one of 25, 465, 587, 2526)
```
2. Install, create an account, and log in
4. Create the first project by following the instructions
5. Delete the project and show the following errors
```
Leantime\Domain\Projects\Services\Projects::getProject(): 
Argument #1 ($id) must be of type int, string given, called in /var/www/html/app/Domain/Menu/Services/Menu.php on line 125 
(View: /var/www/html/app/Domain/Menu/Templates/headMenu.blade.php) 
(View: /var/www/html/app/Domain/Menu/Templates/headMenu.blade.php) 
(View: /var/www/html/app/Domain/Menu/Templates/headMenu.blade.php)
```
![scrnli_3_7_2024_2-05-45 PM](https://github.com/Leantime/leantime/assets/93976113/16da85af-3ad6-459d-be38-ecb4f65b874b)

**Leantime Version**
Latest Docker Image(linux/amd64):
```
leantime/leantime                             latest             757522d2a3f4   8 days ago     771MB
```

**Comment by rimi-itk** [comment]
Might be related to https://github.com/Leantime/leantime/issues/2392.

**Comment by marcelfolaron** [comment]
Fixed in 3.1.0-beta
